cybershell/vulnerability_kb.py

```
@@ -0,0 +1,740 @@
+                applies_to=["all"],
+                success_rate=0.4
+            ),
+        ]
+        
+        self.bypass_techniques.extend(bypass_techniques)
+    
+    def add_payload(self, payload: VulnPayload) -> bool:
+        """Add a new payload to the knowledge base"""
+        try:
+            self.payloads[payload.category].append(payload)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)